src/Microsoft.Dnx.Runtime/Loader/PackageAssemblyLoader.cs

```diff
@@ -28,7 +28,19 @@ public Assembly Load(AssemblyName assemblyName, IAssemblyLoadContext loadContext
         {
             // TODO: preserve name and culture info (we don't need to look at any other information)
             string path;
-            if (_assemblies.TryGetValue(new AssemblyName(assemblyName.Name), out path))
+            var newAssemblyName = new AssemblyName(assemblyName.Name);
```

Comment:
assemblyName is -
`xunit.runner.utility.AspNet, Version=99.99.99.0, Culture=neutral, PublicKeyToken=xxx`

entry in _assemblies is `xunit.runner.utility.AspNet`